README.adoc

```diff
@@ -173,8 +173,9 @@ mvn liberty:stop-server
 
 == Running the services
 
-Before you proceed, build and start the provided `system` and `inventory` services in the starting project.
-Navigate to the `start` directory and run the `install` and `liberty:start-server` Maven goals:
+You'll need to start the services to see basic traces appear in Zipkin. So, before you proceed, build
+and start the provided `system` and `inventory` services in the starting project. Navigate to the `start`
+directory and run the Maven `install` and `liberty:start-server` goals:
```

Comment:
Suggested changes:
Start the services to see basic traces appear in Zipkin. Before you proceed, build...

Reply:
This is what I had it as originally, but writing this paragraph this way makes it feel like a bunch of sentences thrown together. I changed it to this so that it has a nicer flow from one sentenced to the next. Rather than saying "do this, do that", its more like "before you go on, this needs to be done, so do this to get that done". I guess that makes it more conversational too.

Reply:
Sounds good.